packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -191,13 +191,29 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
     protected _attachState: AttachState;
     protected readonly summarizerNode: ISummarizerNodeWithGC;
 
+    /**
+        @deprecated Dummy summary tracker for back compat
+    */
+    private readonly summaryTracker = {
```

Comment:
this is gross, but needed to make the back compat tests pass. can be remove next release